pykafka/simpleconsumer.py

```diff
@@ -646,20 +655,18 @@ def _handle_success(parts):
                         min_bytes=self._fetch_min_bytes
                     )
                 except (IOError, SocketDisconnectedError):
+                    if self._running:
+                        unlock_partitions(iterkeys(partition_reqs))
```

Comment:
Ah, yes, sorry! I need to bring my glasses obviously. No bugs here, all good to go :)